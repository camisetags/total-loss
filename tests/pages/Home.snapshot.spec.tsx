import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import HomePage from '../../src/app/pages/Home';

describe('Home Page', () => {
  it('should render Home page', () => {
    const rendered = renderer.create(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>,
    );
    expect(rendered).toMatchSnapshot();
  });
});
