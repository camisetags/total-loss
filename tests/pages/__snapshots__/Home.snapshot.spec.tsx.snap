// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render Home page 1`] = `
<div>
  <h1
    className="main-title"
  >
     Me drunk 
  </h1>
  <div
    className="center-grid"
  >
    <a
      href="deck-select"
      onClick={[Function]}
    >
      <button
        className="button big"
      >
        Jogar!
      </button>
    </a>
  </div>
</div>
`;
